exporters/ostream/test/ostream_log_test.cc

```diff
@@ -97,24 +97,28 @@ TEST(OstreamLogExporter, DefaultLogRecordToCout)
   std::cout.rdbuf(original);
 
   std::vector<std::string> expected_output{
-      "{\n"
+      "{\n",
       "  timestamp          : 0\n",
-      "  severity_num       : 0\n"
-      "  severity_text      : INVALID\n"
+      "  severity_num       : 0\n",
+      "  severity_text      : INVALID\n",
       "  body               : \n",
       "  resource           : \n",
-      "    telemetry.sdk.version: " OPENTELEMETRY_VERSION "\n",
+      "    telemetry.sdk.version: ",
+      OPENTELEMETRY_VERSION,
+      "\n",
```

Comment:
Works too, but the original code was correct: string concatenation done in the parser.

Reply:
The original code is correct and this aswell. `clang-tidy `warns that there is no comma and the formatting moved them to next line. 

Reverting this.